test/utils/tox/requirements-py3.txt

```diff
@@ -13,3 +13,4 @@ python3-memcached
 python-systemd
 botocore
 boto3
+apache-libcloud
```

Comment:
CI does not use this requirements file. Take a look at https://github.com/ansible/ansible/blob/devel/test/runner/requirements/units.txt for requirements for unit tests.